tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -36,6 +36,21 @@ function createProvider(contextDefs: Record<string, any>) {
   };
 }
 
+export function BaseProviders(props: {
+  children: React.ReactNode;
+  organization?: Organization;
+}): React.ReactElement {
+  return (
+    <CacheProvider value={cache}>
+      <ThemeProvider theme={lightTheme}>
+        <OrganizationContext.Provider value={props.organization ?? null}>
+          {props.children}
+        </OrganizationContext.Provider>
+      </ThemeProvider>
```

Comment:
Will using this with `mountWithTheme` result in two themeproviders in the render tree? Should this provider just do organization prop -> context conversion?

Reply:
Nevermind, just saw that #32066 does what I was suggesting.